src/commands/moderation/prefixCommands/functions/setContent.ts

```diff
@@ -0,0 +1,232 @@
+        .setStyle(TextInputStyle.Short)
+        .setMaxLength(255)
+        .setMinLength(0)
+        .setRequired(false)
+        .setValue(foundContent ? foundContent.title : '');
+
+    const commandContentContent = new TextInputBuilder()
+        .setCustomId('commandContentContent')
+        .setLabel('Content')
+        .setPlaceholder('Provide the content for the command.')
+        .setStyle(TextInputStyle.Paragraph)
+        .setMaxLength(2048)
+        .setMinLength(0)
+        .setRequired(false)
+        .setValue(foundContent && foundContent.content ? foundContent.content : '');
+
+    const commandContentImageUrl = new TextInputBuilder()
+        .setCustomId('commandContentImageUrl')
+        .setLabel('Image URL')
+        .setPlaceholder('Provide an optional Image URL for the command.')
+        .setStyle(TextInputStyle.Short)
+        .setMaxLength(255)
+        .setMinLength(0)
+        .setRequired(false)
+        .setValue(foundContent && foundContent.image ? foundContent.image : '');
+
+    const titleActionRow = new ActionRowBuilder<TextInputBuilder>().addComponents(commandContentTitle);
+    const contentActionRow = new ActionRowBuilder<TextInputBuilder>().addComponents(commandContentContent);
+    const imageUrlActionRow = new ActionRowBuilder<TextInputBuilder>().addComponents(commandContentImageUrl);
+
+    contentModal.addComponents(titleActionRow);
+    contentModal.addComponents(contentActionRow);
+    contentModal.addComponents(imageUrlActionRow);
+
+    await interaction.showModal(contentModal);
+
+    const filter = (interaction: {
+        customId: string;
+        user: { id: any; };
+    }) => interaction.customId === 'commandContentModal' && interaction.user.id;
+
+    let title = '';
+    let content = '';
+    let image = '';
+
+    try {
+        //Await a modal response
+        const modalSubmitInteraction = await interaction.awaitModalSubmit({
+            filter,
+            time: 120000,
+        });
+
+        title = modalSubmitInteraction.fields.getTextInputValue('commandContentTitle').trim();
+        content = modalSubmitInteraction.fields.getTextInputValue('commandContentContent').trim();
+        image = modalSubmitInteraction.fields.getTextInputValue('commandContentImageUrl').trim();
+
+        if (!title && !content && !image) {
+            await modalSubmitInteraction.reply({
+                content: 'You did not provide any content information and the change was not made.',
+                ephemeral: true,
+            });
+            return;
+        }
+        await modalSubmitInteraction.reply({
+            content: 'Processing command content data.',
+            ephemeral: true,
+        });
+    } catch (error) {
+        //Handle the error if the user does not respond in time
+        Logger.error(error);
+        await interaction.followUp({
+            content: 'You did not provide the necessary content information and the change was not made.',
```

Comment:
```suggestion
            content: 'You did not provide the necessary content information in time (2 minutes) and the change was not made.',
```
I would include the information that there is a 2-minute limit in the error message. That way, users get educated about the limitation should they ever come across it.